docs/user-guide/deployments/deploy-on-kubernetes/deploy-greptimedb-standalone.md

```diff
@@ -0,0 +1,262 @@
+---
+keywords: [Kubernetes, deployment, getting started, prerequisites, installation]
+description: Step-by-step guide to deploying a GreptimeDB standalone on Kubernetes.
+---
+
+# Deploy GreptimeDB Standalone
```

Comment:
I think most of the steps of the docs are duplicated with the quick start. Can we make it short? For example, you can refer to the same steps in the quick start.